Jenkinsfile

```diff
@@ -23,9 +23,14 @@ pipeline {
     OS_FLAVOR_NAME=credentials('OS_FLAVOR_NAME')
     OS_IMAGE_ID=credentials('OS_IMAGE_ID')
     GIT_COMMIT_SHORT = sh(
```

Comment:
is `GIT_COMMIT_SHORT` still used ? if not please remove it

Reply:
Yes, see above. It's now only for logz.io